gadget/layout.go

```diff
@@ -99,11 +99,50 @@ type LaidOutStructure struct {
 // meh this function is weirdly placed, not sure what to do w/o making schemaMBR
 // constant exported
 func IsRoleMBR(ls LaidOutStructure) bool {
-	return ls.Role == schemaMBR
+	return ls.Role() == schemaMBR
+}
+
+// These accessors return currently what comes in the gadget, but will use
+// OnDiskVolume data when the latter is made part of LaidOutStructure.
+
+// Type returns the disk type
```

Comment:
this probably needs to be closer to what VolumeStructure.Type comment says. Also it's missing the period at the end.

Reply:
Fixed